modules/ossm-tutorial-kiali-istio-config.adoc

```diff
@@ -0,0 +1,18 @@
+[id="ossm-kiali-tutorial-istio-config_{context}"]
+= Istio Config page
```

Comment:
You're right, these are more like procedures.  They're going away in GA (replaced by *actual* procedures) but for now I'm going with "Exploring [page]" since I want to keep the tutorial for now.